RDF.js

```diff
@@ -983,20 +983,23 @@ function importItem(newItem, node) {
 		n.so + "volumeNumber"], true);
 	
 	// issue
+	var issueNodes = [node];
 	if (container) {
-		newItem.issue = getFirstResults([container, node], [n.prism + "number",
-			n.prism2_0 + "number",
-			n.prism2_1 + "number",
-			n.eprints + "number",
-			n.bibo + "issue",
-			n.dc + "source.Issue",
-			n.dcterms + "citation.issue",
-			n.so + "issueNumber"], true);
-	}
+		issueNodes.unshift(container);
+	}
+	newItem.issue = getFirstResults(issueNodes, [n.prism + "number",
+		n.prism2_0 + "number",
+		n.prism2_1 + "number",
+		n.eprints + "number",
+		n.bibo + "issue",
+		n.dc + "source.Issue",
+		n.dcterms + "citation.issue",
+		n.so + "issueNumber"], true);
+	
 
 	// these mean the same thing
-	newItem.patentNumber = newItem.number = newItem.issue;
-	
+	newItem.patentNumber = newItem.reportNumber = newItem.number = newItem.issue;
```

Comment:
This is fine temporarily, but if `newItem.number` isn't setting `reportNumber` or `patentNumber` in their respective item types, that's a bug. It should automatically use base field mappings. (`issue` is a separate field.)

Reply:
I think that they are set also automatically. I will check that again and then delete the unnecessary assignments.